nautobot/core/tests/test_views_utils.py

```diff
@@ -219,3 +227,286 @@ def test_anonymous_user_get_shared_views_only(self):
         self.assertEqual(saved_views.count(), 2)
         expected_names = ["saved_view_shared", "saved_view_shared_different_owner"]
         self.assertEqual(list(saved_views.values_list("name", flat=True)), expected_names)
+
+
+class GetBulkQuerysetFromViewTestCase(TransactionTestCase):
+    """Class to test get_bulk_queryset_from_view with VRF."""
+
+    def setUp(self):
+        super().setUp()
+        self.user = User.objects.create_user(username="testuser", password="testpass", is_superuser=True)  # noqa: S106  # hardcoded-password-func-arg -- ok as this is test code only
```

Comment:
Shouldn't be needed - `super.setUp()` should already create a `nautobotuser` account as `self.user`?